versioned_docs/version-v0.72/node-operators/how-to/solve-frequent-issues.md

```diff
@@ -182,3 +182,19 @@ AutoInitialiseFromNetworkHistory = true
 maxNumberOfFirstConnectionRetries = 1800
 ```
 8. Start `vegavisor` or `core + date node` (order doesn't matter)
+
```

Comment:
Should we change filepath to `version-v0.73` ?

Reply:
When there is a 0.73 on mainnet, a folder with that name will come to exist. Also this file path is only visible inside the repo, not on the website.